spk/hassio/Makefile

```diff
@@ -0,0 +1,45 @@
+SPK_NAME = hassio
+SPK_VERS = $(shell date +%Y%m%d)
```

Comment:
Please create your own package versioning then. Because this way prevents "reproductable" build.